constants/__init__.py

```diff
@@ -21,12 +23,14 @@
 )
 
 from competitions.data import CompetitionId
+from finetune.datasets.ids import DatasetId
+from finetune.eval.method import EvalMethodId
 
 # ---------------------------------
 # Project Constants.
 # ---------------------------------
 
-__version__ = "2.3.0"
+__version__ = "2.4.0"
 version_split = __version__.split(".")
 __spec_version__ = (
     (1000 * int(version_split[0]))
```

Comment:
Need to bump the state version here too right? At least this time. In the future the eval task hash helper will resolve it.

Reply:
Related: Did you want to include the check against the eval tasks changing to clear the tracker for the eval results?

Reply:
Bumped the vali state version.

The modeltracker clearing is in validator.py